test/units/testsuite-07.sh

```diff
@@ -2,30 +2,30 @@
 set -eux
 set -o pipefail
 
->/failed
+: >/failed
 
-cat <<'EOL' >/lib/systemd/system/my.service
+cat >/lib/systemd/system/my.service <<EOF
```

Comment:
Using quotes around the terminator meant that expansion was not performed in the body. But there wasn't anything to expand. So this change seems to be correct, i.e. to have no effect. I'm just wondering why the original pattern was used... Maybe it's better not to ask.